### PR TITLE
LPS-120800 Baseline

### PR DESCRIPTION
**Pull request by marcelabc**

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

File: modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/resource/v2_0/DataDefinitionResourceImpl.java [hunk 1]
```diff
@@ -1608,6 +1608,17 @@ private DataDefinition _toDataDefinition(DDMStructure ddmStructure)
...
 		}
 
+		if (ddmFormValidationException instanceof
+				DDMFormValidationException.MustSetValidType) {
+
+			DDMFormValidationException.MustSetValidType mustSetValidType =
+				(DDMFormValidationException.MustSetValidType)
+					ddmFormValidationException;
+
+			return new DataDefinitionValidationException.MustSetValidFieldType(
```
Comment (jeyvison) [inline modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/resource/v2_0/DataDefinitionResourceImpl.java:1618]: Hey Marcela, can you please Match the name pf the exceptions to MustSetValidType?

**Review by jeyvison — CHANGES_REQUESTED** [review]

**Pull request closed without merging**